packages/wp-story-editor/src/api/media.js

```diff
@@ -180,17 +184,24 @@ export function uploadMedia(config, file, additionalData) {
     web_stories_is_muted: isMuted,
     post: templateId || storyId || mediaId,
     original_id: originalId,
-    web_stories_trim_data: trimData,
-    web_stories_base_color: baseColor,
-    web_stories_blurhash: blurHash,
+    meta: {
+      web_stories_base_color: baseColor,
+      web_stories_blurhash: blurHash,
+      web_stories_cropped_origin_id: originalCropId,
```

Comment:
"origin" is a good word, or perhaps "parent"

Can we rename `originalCropId` accordingly?

It's confusing to see `originalId` and `originalCropId` next to eachother.

Reply:
What name do you like most?

Reply:
Maybe `cropOriginId`?